examples/classic/docusaurus.config.js

```diff
@@ -13,6 +13,11 @@ const config = {
   title: 'My Site',
   tagline: 'Dinosaurs are cool',
   favicon: 'img/favicon.ico',
+  siteConfig: {
+      markdown: {
+        onUnusedDirective: 'throw'
+      }
+  },
```

Comment:
example sites are generated and shouldn't be modified

Reply:
I added this to test my changes. I will remove these eventually. I was looking for help with testing my changes for the screenshot I have attached.

Reply:
You can test your changes on our own website in `website/docusaurus.config.ts`